mesonbuild/mtest.py

```diff
@@ -128,6 +138,10 @@ def add_arguments(parser: argparse.ArgumentParser) -> None:
                         help='Only run tests belonging to the given suite.')
     parser.add_argument('--no-suite', default=[], dest='exclude_suites', action='append', metavar='SUITE',
                         help='Do not run tests belonging to the given suite.')
+    parser.add_argument('--exclude', default=[], dest='exclude_tests', action='append', metavar='TEST',
```

Comment:
I think it could worth having a short option `-x` as well.

Reply:
I was not certain which letter to assign. I checked few other tools and [`pytest`](https://docs.pytest.org/en/7.2.x/reference/reference.html#command-line-flags) just uses `-k "not test_name", [PHPUnit](https://docs.phpunit.de/en/9.5/textui.html) or [cargo](https://doc.rust-lang.org/cargo/commands/cargo-test.html) do not appear to have anything like that.